### PR TITLE
added eu-west-3 AWS region 

### PR DESCRIPTION
**Pull request by keremdokuz**
Added eu-west-3 (Paris) to the available AWS regions.

Include a provider and a storage driver for the AWS region eu-west-3. This PR will allow working with data located in the eu-west-3 region of AWS using libcloud.

- done, ready for review

- [x] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [x] Documentation
- [x] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [x] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

**Comment by Kami** [comment]
@keremdokuz Thanks for the contribution.

Can you please fix failing lint checks? Besides that, change LGTM.

File: libcloud/storage/drivers/s3.py [hunk 1]
```diff
@@ -1435,6 +1395,17 @@ class S3EUWest2StorageDriver(S3StorageDriver):
     region_name = "eu-west-2"
 
 
+class S3EUWest3Connection(S3SignatureV4Connection):
```
Comment (Kami) [inline libcloud/storage/drivers/s3.py]: Those two classes are actually not needed anymore - this was an old approach (class per region) which is not used anymore and has been replaced via `region` class constructor argument.

Can you please remove them? Thanks.

Comment (keremdokuz) [reply libcloud/storage/drivers/s3.py]: I removed them :slightly_smiling_face:

**Review by Kami — APPROVED** [review]

**Comment by Kami** [comment]
Merged into trunk. Thanks.

**Comment by keremdokuz** [comment]
> Merged into trunk. Thanks.

@Kami are you planning on a release in the near future?

**Comment by Kami** [comment]
@keremdokuz We don't have a fixed release cadence, but if I get a chance, I will try to prepare a new release some time in the next couple of weeks.

**Pull request merged**